frontend/src/components/VCSSetupWizard.vue

```diff
@@ -72,18 +67,25 @@ export default {
     VCSProviderConfirmPanel,
   },
   setup() {
+    const { t } = useI18n();
     const store = useStore();
     const router = useRouter();
 
+    const stepList: BBStepTabItem[] = [
+      { title: t('version-control.setting.add-git-provider.basic-info.self') },
+      { title: t('version-control.setting.add-git-provider.oauth-info.self') },
+      { title: t('common.confirm') },
+    ];
+
     const state = reactive<LocalState>({
       config: {
         type: "GITLAB_SELF_HOST",
-        name: "GitLab self-host",
+        name: t('version-control.setting.add-git-provider.gitlab-self-host'),
         instanceUrl: "",
         applicationId: "",
         secret: "",
       },
-      currentStep: 0,
+      currentStep: 1,
```

Comment:
Do we change it by design?

Reply:
Good eye. 

Clearly an awful mistake... Was used for quickly testing different panel